pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -53,7 +54,7 @@
 
 , passthru ? {}
 
-, doCheck ? false
+, doCheck ? config.doCheckByDefault or false
```

Comment:
I will set this to `true` in a separate PR. The builder still needs `doCheck` though, because python's `doCheck` is stdenv's `doInstallCheck`.

Reply:
This should have the same default as that for stdenv. The format-specific bits can change the default.

Reply:
> The builder still needs doCheck though, because python's `doCheck` is stdenv's `doInstallCheck`.

Ah that's what I was missing---makes sense now. Want to add a comment?

Reply:
It already has such a comment in a file there.

Reply:
Yeah I suppose the remaining is evidence enough.